xml/System.IO.Enumeration/FileSystemEnumerable`1.xml

```diff
@@ -35,7 +35,17 @@
   <Docs>
     <typeparam name="TResult">The type that this enumerable encapsulates.</typeparam>
     <summary>Allows utilizing custom filter predicates and transform delegates for enumeration purposes.</summary>
-    <remarks>To be added.</remarks>
+    <remarks>
+    <format type="text/markdown"><![CDATA[
+
+## Examples
+
+The following example shows how to use the <xref:System.IO.Enumeration.FileSystemEnumerable`1> constructor with basic filtering options:
+
+[!code-csharp[FileSystemEnumerable#1](~/samples/snippets/csharp/VS_Snippets_CLR/IO.Enumeration.FileSystemEnumerable/CS/Sample1.cs#1)]  
```

Comment:
I am not sure what the `#1` does, but I noticed all existing examples have a similar suffix. I think it's the anchor, right? I'll verify it in the preview.

Reply:
The preview shows the code example section empty, probably due to the unnecessary #1. Removing it to see if that helps:
```suggestion
[!code-csharp[FileSystemEnumerable](~/samples/snippets/csharp/VS_Snippets_CLR/IO.Enumeration.FileSystemEnumerable/CS/Sample1.cs)]  
```